packages/reactive-dict/reactive-dict.js

```diff
@@ -94,19 +103,19 @@ _.extend(ReactiveDict.prototype, {
     // XXX we could allow arrays as long as we recursively check that there
     // are no objects
     if (typeof value !== 'string' &&
-        typeof value !== 'number' &&
-        typeof value !== 'boolean' &&
-        typeof value !== 'undefined' &&
-        !(value instanceof Date) &&
-        !(ObjectID && value instanceof ObjectID) &&
-        value !== null)
+      typeof value !== 'number' &&
```

Comment:
I'm saying even if parts of the file don't meet the style guide, we shouldn't change them as part of this PR, in which case no prettifier will do the right thing.